build-source-tarball.sh

```diff
@@ -59,6 +59,7 @@ fi
 
 export SCRIPT_ROOT="$(cd -P "$( dirname "$0" )" && pwd)"
 SDK_VERSION=$(cat $SCRIPT_ROOT/DotnetCLIVersion.txt)
+DARC_DLL="$SCRIPT_ROOT/tools-local/arcade-services/artifacts/bin/Microsoft.DotNet.Darc/Release/netcoreapp2.1/Microsoft.DotNet.Darc.dll"
```

Comment:
Do we build darc to this location?  Do we need to?  Can we just install it?  My naive thought is that we don't need it in the tarball, so we don't need the source for it.

Reply:
I don't think we really need to build Darc, however, Darc is not published very often and the published version is currently quite far behind.  I will check on whether this is planned to change anytime soon, otherwise I think we need to stick with it for now.

Reply:
Logged https://github.com/dotnet/source-build/issues/1041 for this.